spec/features-json/user_json_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "spec_helper"
+require "app/features-json/user_json_controller"
+require "app/services/user_service"
+
+describe FastlaneCI::UserJSONController do
+  def app
+    described_class
+  end
+  let(:json) { JSON.parse(last_response.body) }
+
+  before do
+    allow(FastlaneCI.dot_keys).to receive(:encryption_key).and_return("test")
+  end
+
+  describe "/api/user" do
```

Comment:
How does this test work, given that we disable network requests via Octokit?

Reply:
Investigating this now

Reply:
Ha, it actually doesn't work on my machine, fixing now

Reply:
Ah because I removed the `email` parameter last minute, we manually passed one before